Ingress/controllers/nginx-third-party/examples/custom-nginx-conf.json

```diff
@@ -0,0 +1,29 @@
+{
+  "metadata": {
+    "name": "prodNginxConf"
+  },
+  "apiVersion": "k8s.io/stable/v1",
```

Comment:
It would be nice if this was yaml, and you added a short comment above each one. Something like:
```
// Minimum is TLS 1.1 because 1.0 suffers a downgrade attack. 
sslProtocols

// Connection keepalive. Higher keepalive usually means fewer connections. 
// See https://www.nginx.com/blog/http-keepalives-and-web-performance/ for more details. 
keepAlive
```
1.0 is BEASTed right? or am i confusing them.